### ISSUE TITLE
Test and fix the local installation guides

### ISSUE THREAD

**Issue by andaag** [kubeflow/website#1097]
So I've been trying to get kubeflow up and running, to play with it as a dev. I have kubernetes experience as a user, as we (like very many other companies I can imagine) have a team that takes care of kubernetes. I'm not going to ask them to set this up until I can compare it with other solutions, and I imagine that's a fairly common case.

So far I've tried 4 different install instructions, and not one of them solved my simple case of "just testing this locally".

1. I started with microk8s.
- My first roadblock was due to network devices, microk8s creates network devices that needs to be whitelisted in iptables.
- Next roadblock was the dynamic provisioner
- After that something was still having connection problems with mysql. I'm guessing I setup the dynamic provisioning incorrectly. At this point I figured I was doing something basic wrong, because this part should just work, so I moved on to another guide.

2. Then I tried multipass
- These scripts whitelist the required tables by default when using microk8s I realized. Of course I found later in the process.
- Then the scripts crash on "ERROR find objects: lib/ksonnet-lib/v1.15.2/k8s.libsonnet:17681:93-94 Unknown variable: x". I'm not going to paste details for this as it's incredibly easy to reproduce.
- I'm guessing because this is running an ancient version.

3. Then I looked at minikf
- This comes in video format, and I immediately skipped it.

4. Then I tried running it up on AWS.
- This was fairly painless, until you get to the point where you want to access things.
- I tried using kubeflow port-forward, that gave me some random errors on the page. I'm guessing because it's not the intended way.
- The install instructions at this point tell you how to get single sign on manually working through aws, and registering a domain name. Neither of which I was in the mood to do (at this point I was also fairly fed up. I suspect had I started on this it would have worked just fine).

## My very strong suggestion:
Make one install instruction for testing this locally, test it against the latest versions and keep it up to date. You don't need to have instructions for every single combination of system on the main page, especially when that degrades in quality and they are not maintained.

I had set off half a day to play with kubeflow from a user perspective, and if that part looked good I'd ask someone who is a kubernetes admin to set this up serverside for me. Instead I spent half a day debugging things that I will never need to know/debug in the future. It was extremely frustrating, and me running from install instructions to install instructions, some of them anciently out of date, does not make me feel like kubeflow is a stable good product. 

My next step would probably be minikube. Not because I want to use minikube, but because I randomly just realized the documentation there is refering to 0.6.1. Which makes me hopeful it's written recent enough that it's up to date.

**Comment by sarahmaddox** [comment]
Here's the issue requesting an update to the Microk8s docs: https://github.com/kubeflow/website/issues/1113

**Comment by sarahmaddox** [comment]
Thanks for testing and reporting the getting-started guides @andaag. We've done a fair bit of work recently to improve the getting-started experience.

I've streamlined the getting-started process as a whole in PR https://github.com/kubeflow/website/issues/1421.

Thanks also for your feedback on the MicroK8s experience in particular, on issue https://github.com/kubeflow/website/issues/1694.

We've also removed the page on Minikube, because testing and feedback shows that this method of using Kubeflow on Minikube is no longer feasible. 

The MiniKF team are testing MiniKF and will update the doc for Kubeflow v1.0 as soon as they can.

Based on the above, I'm closing this issue now. We can raise further issues as they occur.

/close
